src/index.js

```diff
@@ -19,11 +38,39 @@ export default class Rollup extends Plugin {
     this.rollupOptions  = options.rollup || {};
     this._lastBundle = null;
     this._output = null;
+    this.stagingPath = path.join(tmpdir(), `broccoli-rollup-staging`);
+    rimraf(this.stagingPath);
```

Comment:
I can stat the path instead, otherwise L43 will get an EXIST. I also don't know if this has the potential of colliding between projects because the path is going to be stable.